imblearn/tests/test_pipeline.py

```diff
@@ -1073,3 +1076,18 @@ def test_pipeline_fit_then_sample_3_samplers_with_sampler_last_estimator():
     X_fit_then_sample_res, y_fit_then_sample_res = pipeline.sample(X, y)
     assert_array_equal(X_fit_sample_resampled, X_fit_then_sample_res)
     assert_array_equal(y_fit_sample_resampled, y_fit_then_sample_res)
+
+
+def test_make_pipeline_memory():
+    cachedir = mkdtemp()
+    if LooseVersion(joblib_version) < LooseVersion('0.12'):
+        # Deal with change of API in joblib
+        memory = Memory(cachedir=cachedir, verbose=10)
```

Comment:
I would not bother. I would make the one that work and update when the test will break when we will update the scikit learn version